stable/prisma/templates/deployment.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "prisma.fullname" . }}
+  labels:
+    app: {{ template "prisma.name" . }}
+    chart: {{ template "prisma.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: {{ template "prisma.name" . }}
+      release: {{ .Release.Name }}
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: {{ template "prisma.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: prisma
+              containerPort: 4466
+              protocol: TCP
+          env:
+            - name: CONFIG
+              valueFrom:
+                configMapKeyRef:
+                  name: {{ template "prisma.fullname" . }}
+                  key: config
+            {{- if .Values.secret }}
+            - name: PRISMA_API_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "prisma.fullname" . }}
+                  key: api-secret
+            {{- end}}
+            - name: PRISMA_CONFIG_PATH
+              value: "/app/config.yml"
```

Comment:
Where is this file created? Did I missed something?

Reply:
It's created by the `prerun_hook.sh` script that gets executed on the Prisma server starts.

Reply:
Oh, yes you're using the heroku version. Is this version documented somewhere?

Reply:
Oh okay, I was curious to see how they implemented the heroku one. You find it in Prisma sources?

Reply:
I heard about it from a Prisma open source user. Then I dig in around the container by myself to figure it out.